docs/app/pages/components/components-sections/file-upload/file-upload/file-upload.component.html

```diff
@@ -0,0 +1,59 @@
+<label for="file-upload-demo" ng2FileDrop [class.file-upload-dropping]="fileOver" (fileOver)="fileOver = $event" [uploader]="uploader" class="file-upload-dropzone">
+    Click here or Drag and Drop to upload files
+</label>
+
+<input type="file" id="file-upload-demo" class="sr-only" ng2FileSelect [uploader]="uploader" multiple>
+
+<ux-progress-bar [value]="uploader.progress ? uploader.progress : 0"></ux-progress-bar>
+
+<p class="file-upload-progress-indicator">{{ uploader.progress ? uploader.progress : 0 }}%</p>
+
+<ul *ngIf="uploader.queue.length" class="file-upload-queue">
+    <li *ngFor="let item of uploader.queue">
+        <a (click)="item.remove()">
+            <span>{{ item.file.name }}</span>
+            <span class="file-upload-queue-remove hpe-icon hpe-close"></span>
+        </a>
+    </li>
+</ul>
+
+<button type="button" class="btn button-primary m-r-xs" (click)="uploader.uploadAll()" [disabled]="!uploader.getNotUploadedItems().length">
```

Comment:
Could you wrap these in `.btn-container` - this is our flex container for buttons to ensure exact spacing between them.